### PR TITLE
Fix notification bar line height breaking container. Fixes #525

### LINKED ISSUE #525: [bug] Notification bar line height breaking container.
## Description

The [line-height](https://github.com/mozilla/protocol/blob/master/src/assets/sass/protocol/components/_notification-bar.scss#L30-L31) is knocking longer text out of it's container at certain screen sizes.

Screen Shot 2019-11-26 at 3 23 52 PM
Screen Shot 2019-11-26 at 3 23 44 PM

### PR DESCRIPTION
**Pull request by tikhsuP**
## Description
 Fix notification bar line height breaking container.

### Issue
#525

### CODE REVIEW

File: src/assets/sass/protocol/components/_notification-bar.scss [hunk 1]
```diff
@@ -20,16 +20,11 @@
...
     }
 
     p {
         display: inline-block;
-        margin: 0 15px;
-
-        @media #{$mq-sm} {
-            line-height: 46px;
-        }
+        margin: 12px 45px;
```
Comment (craigcook) [inline src/assets/sass/protocol/components/_notification-bar.scss]: We should use spacing tokens for these values. I'd suggest `margin: $spacing-md $spacing-2xl;` (which is 16px and 48px)

Comment (tikhsuP) [reply src/assets/sass/protocol/components/_notification-bar.scss]: Thanks for your guidance @craigcook 
I have made the necessary changes.

**Comment by tikhsuP** [comment]
While fixing this issue, I encountered a slight inconsistency(bug) in the layout that will follow.
## Description
When the text message between the paragraph tags is long enough to push the anchor tags to the next line then we can see a margin of 0px between notification message and the bottom of notification bar which damages the look.

### Actual result
Screenshot from 2020-02-15 19-58-39

### Expected result
Screenshot from 2020-02-15 20-23-49

If this is significant and needs to be corrected then I seek your permission to open an issue to fix.

**Comment by craigcook** [comment]
I see what you mean. So on further thought, I think it would probably be better to use padding on the container rather than margins on the contents. That will ensure adequate spacing inside the box, regardless of what's inside it.

**Comment by tikhsuP** [comment]
Yes @craigcook, that would be more appropriate.
I have made the necessary changes, please check.

###Result

Screenshot from 2020-02-20 23-59-37

Thanks for your guidance.

**Comment by craigcook** [comment]
Looks great! 

One last request: please add a line to CHANGELOG.md under `## HEAD` noting the change, with a reference to the issue number (#525). You'll probably want to rebase first to make sure you have the latest version. Then we'll be all set to merge this at last. Thanks for your patience. Nice work!

**Comment by craigcook** [comment]
I rebased to resolve the conflicts, updated the changelog, and merged this fix manually in 31e0d1ea576c874ac26a7809b3f47296296426aa

Thanks for contributing, and sorry it took so long!

**Pull request closed without merging**